spec/factories/tasks.rb

```diff
@@ -5,19 +5,22 @@
     deadline { Time.current }
     status { 2 }
     priority { 1 }
+    user
   end
   factory :testtask_second, class: Task do
     title { 'factory_title2' }
     content { 'factory_content2' }
     deadline { Time.current.next_month }
     status { 0 }
     priority { 0 }
+    user
   end
   factory :testtask_third, class: Task do
     title { 'factory_title3' }
     content { 'factory_content3' }
     deadline { Time.current.tomorrow }
     status { 1 }
     priority { 2 }
```

Comment:
こちらのenumの値については、Model側では英語で定義することが一般的です。
英語で定義することによって、値を明示して設定できます。
また、Task.high といった形でenum名のメソッドが利用できるようになります。

今回は途中まで日本語で作ってしまったので、修正が大変であればこのままでも大丈夫です。
```suggestion
    priority { :high }
```

Reply:
enum利用についてあらためて学習いたします。
今回はこのまま/修正なしにできればと思っています。